### PR TITLE
[Optimize] Support send batch parallelism for olap table sink

### LINKED ISSUE #6398: [Optimize] Sometimes data serialization time may be long when wait to send data to tablet host
We found that when do olap table sink,   data serialize may cost a lot of time, but here is only one single thread for data serialization, we can set parallelism for it to accelerate the speed.

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [x] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6398 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
How much is this performance improvement?

**Comment by caiconghui** [comment]
> How much is this performance improvement?

This is the first step you speed up olap table sink. and the bottleneck for olaptable sink is still need to be deep solved.
I make a test in my environment with 5 node in cluster

MySQL [test]> insert into info_p1 select * from info_p1;
ERROR 5024 (HY000): errCode = 2, detailMessage = Execute timeout
MySQL [test]> set send_batch_parallelism=1;
Query OK, 0 rows affected (0.00 sec)

MySQL [test]> set send_batch_parallelism=5;
Query OK, 0 rows affected (0.00 sec)

MySQL [test]> insert into info_p1 select * from info_p1;
Query OK, 84272656 rows affected (3 min 24.68 sec)
{'label':'insert_a88e439072c74d4b-927690bb54979af0', 'status':'VISIBLE', 'txnId':'76736'}

**Review by morningman — COMMENTED** [review]
I think for any load job, if the user does not explicitly set the send_batch_parallelism parameter, the value of the session variable will be used by default.

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -124,7 +126,8 @@ void PInternalServiceImpl<T>::tablet_writer_add_batch(google::protobuf::RpcContr
             }
             st.to_protobuf(response->mutable_status());
         }
-        response->set_execution_time_us(execution_time_ns / 1000);
+        response->set_execution_time_us(execution_time_ns / NANOS_PER_MICRO);
+        response->set_wait_execution_time_us(execution_time_ns / NANOS_PER_MICRO);
```
Comment (morningman) [inline be/src/service/internal_service.cpp:130]:
```suggestion
        response->set_wait_execution_time_us(wait_execution_time_us / NANOS_PER_MICRO);
```

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**